src/main/index.ts

```diff
@@ -126,24 +129,64 @@ Menu.setApplicationMenu(kangarooMenu(KANGAROO_FILESYSTEM));
 app.whenReady().then(async () => {
   SPLASH_SCREEN_WINDOW = createSplashWindow();
   ipcMain.handle("sign-zome-call", handleSignZomeCall);
-  ipcMain.handle('exit', () => {
+  ipcMain.handle("exit", () => {
     app.exit(0);
   });
+
+  // Check for updates
+  if (app.isPackaged && KANGAROO_CONFIG.autoUpdates) {
+    autoUpdater.allowPrerelease = true;
+    autoUpdater.autoDownload = false;
+
+    let updateCheckResult: UpdateCheckResult | null | undefined;
+
+    try {
+      updateCheckResult = await autoUpdater.checkForUpdates();
+    } catch (e) {
+      console.warn('Failed to check for updates: ', e);
```

Comment:
This could be a nit pick, but seems like there's inconsistent use of double and single quote strings, maybe some format configs might fix this

Reply:
yes, right. But I think I'm gonna do this in a separate PR